qotd/blog.md

```diff
@@ -0,0 +1,249 @@
+# Motivational QOTD with Courier and GPT2
+
+## Background
+
+Motivational quotes were rage back in the day with MMS & email forwards were a thing. I remember my parents forwarding me one every morning so that I can start my day with one. Fast forward to today, if you are lucky, you are part of some forward group on your messaging app of choice (Whatsapp, Telegram etc.). 
+
+Inspired by the same idea, today we are going to build a service that sends our friends and family an AI generated motivational quote of the day. Rather than hardcoding a list of motivational quotes, we are going to use a machine learning model to generate a quote on demand. That way we know that we'll never run out of quotes to share!
+
+![what we will build](samples/email.png)
+
+## Instructions
```

Comment:
Are there any pre-reqs required to build this? Level of coding experience needed?

Reply:
Nope, no per-requisites needed to follow along this tutorial.